src/Visualization/Python/PlotPowerMonitors.py

```diff
@@ -57,6 +57,7 @@ def plot_power_monitors(
     domain: Union[Domain[1], Domain[2], Domain[3]],
     dimension_labels: Sequence[str] = [r"$\xi$", r"$\eta$", r"$\zeta$"],
     element_patterns: Optional[Sequence[str]] = None,
+    skip_filtered_modes: int = 0,
     figsize: Optional[Tuple[float, float]] = None,
 ):
     plot_over_time = obs_id is None
```

Comment:
Unrelated to this PR, but we should move the doc string from the `_command` function to this function.